packages/widget/src/components/address-input/address-input.ts

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html } from 'lit';
+import type { HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { ethers } from 'ethers';
+import { validatePolkadotAddress } from '../../utils';
+import { styles } from './styles';
+
+@customElement('sygma-address-input')
+export class AddressInput extends LitElement {
+  static styles = styles;
+  @property({
+    type: String
+  })
+  address: string = '';
+
+  @property()
+  handleAddress?: (address: string) => void;
+
+  @property({
+    type: Object
+  })
+  network?: 'substrate' | 'evm';
```

Comment:
@mpetrunic @sztok7 here I would like to use `Network` enum from `sygma-sdk` but somehow changing this, and then passing network property as `${Network.EVM}` on the test, gives error. I'm going to check that tomorrow. Question here: should the input be aware of the network to branch on validation?

Reply:
I changed to `string` to test implementation. This is the error that gives me when using the enum:

```
Error: Cannot find module '/Users/wainola/Documents/js/chainsafe/sygma-widget/node_modules/@buildwithsygma/sygma-sdk-core/dist-esm/localConfig' imported from /Users/wainola/Documents/js/chainsafe/sygma-widget/node_modules/@buildwithsygma/sygma-sdk-core/dist-esm/config.js
```

Which is weird since `localConfig` or event `config` don't have anything to do with the importing of the enum from the SDK

Reply:
ah, thats bug on sygma-sdk. There is a lot of places which are missing .js extension in import statements.
Issue: https://github.com/sygmaprotocol/sygma-sdk/issues/326 I'll fix that today^^

Reply:
yeah that's the only diff with the other imports from `config` file

Reply:
new PR adding small fixing that was appearing on test [pr](https://github.com/sygmaprotocol/sygma-sdk/pull/329)